Library/Homebrew/cmd/update-report.rb

```diff
@@ -22,16 +22,23 @@ def update_report
     HOMEBREW_REPOSITORY.cd do
       analytics_message_displayed = \
         Utils.popen_read("git", "config", "--local", "--get", "homebrew.analyticsmessage").chuzzle
+      cask_analytics_message_displayed = \
+        Utils.popen_read("git", "config", "--local", "--get", "homebrew.cask.analyticsmessage").chuzzle
```

Comment:
How about just `caskanalyticsmessage`

Reply:
Sure, I was just prepending the other variable with cask to keep some consistency.